.travis.yml

```diff
@@ -6,7 +6,7 @@ addons:
     packages:
       - libnode-dev
 r_github_packages:
-  - mrc-ide/dust
+  - mrc-ide/dust@i87-rng-seed
```

Comment:
NB: Merge https://github.com/mrc-ide/dust/pull/88 first, then remove this before merging here